src/modules/new-request-form/fields/textarea/TextArea.tsx

```diff
@@ -19,6 +20,18 @@ interface TextAreaProps {
   onChange: (value: string) => void;
 }
 
+const StyledField = styled(GardenField)`
+  .ck.ck-editor {
```

Comment:
Could this be added from above as `margin-bottom` so we wouldn't need to target `ck-editor` classes directly?

Reply:
This implementation is coherent with the one used for the Garden `Textarea`, where the margin-top is added to textarea